src/shared/model/Routes.ts

```diff
@@ -12,6 +17,7 @@ export const ValidRoutePathsArray = [
 	'/agree/GRS',
 	'/consents',
 	'/consents/:page',
+	'/consents/our_content',
```

Comment:
our general convention is to use `-` instead of `_` in urls, so if possible would be great to refactor this to 
```suggestion
	'/consents/our-content',
```

Reply:
Yes, I can change that 👍